@uportal/user-profile-menu/README.md

```diff
@@ -0,0 +1,132 @@
+# User Profile Menu
+
+[![NPM Version](https://img.shields.io/npm/v/@uportal/user-profile-menu.svg)](https://www.npmjs.com/package/@uportal/user-profile-menu)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.webjars.npm/uportal__user-profile-menu/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.webjars.npm/uportal__user-profile-menu)
+[![Build Status](https://travis-ci.org/uPortal-contrib/uPortal-web-components.svg?branch=master)](https://travis-ci.org/uPortal-contrib/uPortal-web-components)
+
+## Development
+
+```bash
+# clone the repository if you haven't already
+git clone https://github.com/uPortal-contrib/uPortal-web-components
+
+# navigate to the waffle menu folder
+cd uPortal-web-components/@uportal/user-profile-menu
+
+# install the goods
+npm install
+
+# start the app
+npm start
+```
+
+## Build
+
+Pretty simple:
+
+`npm run build`
+
+This will compile your JavaScript in to `build/static/js/main.{buildnumbers}.js`
+
+## Usage
+
+After building, grab the compiled javascript files and place it on your page.
+You can now place `<user-profile-menu>` anywhere on your page.
+
+Here's the guts of that html file:
+
+```html
+<user-profile-menu>
+  <div slot="header">
+    User Profile
+  </div>
+  <div slot="default">
```

Comment:
If this slot is the default, do we even need to say `slot="default"`?  @ChristianMurphy?

Reply:
It is not required.
anything not given a `slot=""` will be put in the default slot.